docs/pages/blog/2020.md

```diff
@@ -0,0 +1,115 @@
+The material-ui.com marketing website will soon wear a brand new style! We need to create our own identity, distinct from Material Design.
+
+### Material-UI X
+
+We have started to deliver advanced React components in 2020 with the data grid, including a [commercial version](/components/data-grid/#commercial-version) and the beginning of a [date range picker](https://next.material-ui.com/components/date-range-picker/).
+We will double down on these existing components as long as necessary to have them find the market.
+
+By the end of 2021, we aim to have released these components as stable, implement all the [features planned](/components/data-grid/getting-started/#feature-comparison), and at least double the size of the team.
+
+### Material-UI v5
+
+We will release the next major iteration of the library. A highlight of the key improvements coming ✨:
+
+- Polish and promote most of the components that were in the lab in v4 to the core.
+- A new styled engine. Migrate from JSS to emotion (default) and styled-components's `styled` API.
+- Improve customizability. Add new powers to the theme with dynamic color & [variant](https://next.material-ui.com/customization/typography/#adding-amp-disabling-variants) support. Add a new [`sx` prop](https://next.material-ui.com/system/basics/) for quick customizations to all the components. Expose global class names. Deprecate the `makeStyles` and `withStyles` API.
+- Breaking changes on the API to make it more intuitive.
+- Add full support for React strict mode. We recommend to enable it.
+- Improve the performance of the system by a x3-x5 [factor](https://github.com/mui-org/material-ui/issues/21657#issuecomment-707140999).
+- Reduce bundle size: split IE11 into a different bundle [-6kB](https://github.com/mui-org/material-ui/pull/22814#issuecomment-700995216), migrate to emotion [-5kB](https://github.com/mui-org/material-ui/pull/23308#issuecomment-718748835), Popper.js v2 upgrade [-700B](https://github.com/mui-org/material-ui/pull/21761#issuecomment-657135498).
+- Improve the documentation website: search shortcut, page rating, fast material icons copy button, etc.
+
+And [much more](https://github.com/mui-org/material-ui/issues/20012).
+
+### A second theme
+
+While Material Design is a predominant design choice for building rich client-side applications, it's not in the landscape of landing and marketing pages.
+This use case for React has yet to win a large-scale adoption by the community but the traction of Next.js is a strong signal that it has the potential to happen. React pushing [server-side components](https://reactjs.org/blog/2020/12/21/data-fetching-with-react-server-components.html), with zero bundle size overhead could accelerate it.
+
+At the same time, the design trend of the latest hot startups isn't close to Material Design.
```

Comment:
I doubt there is a downward trend with Material Design. It seems to be stable.

https://www.shutterstock.com/ is the biggest flagship I know of using Material Design for a high traffic landing page like website. I put e-commerce, blog, portfolio in the same category.

Reply:
I doubt there is a downward trend with Material Design.

The point is that since Material Design never had much of a place on the home page ("not in the landscape" in your words), it shouldn't come as a surprise that "the design trend of the latest hot startups isn't close to Material Design", since it never was, still isn't, and never will be.